### ISSUE TITLE
Please add support for specials

### ISSUE THREAD

**Issue by Ninelpienel** [RickDB/PlexAniSync#230]
Hey, I read that you don't want to support specials because various sites have different entries for them, but I have my own order and would like to assign the custom mappings manually myself. Can you add that specials can be activated in the settings and then it's possible to map them as season 0?

**Comment by ShanaryS** [comment]
I agree with this feature. If it doesn't get implemented there is an easy way to enable it. I'm doing this on docker but if you're running the script directly, just fork the repo and make the changes there and periodically sync the fork.

1. Download a copy of the custom_mappings_schema.json in the root of the project. In the copy, change the season properties minimum to 0 from 1. This allows reading of season 0 in the custom_mappings.yaml file. Copy it to the docker container: `docker cp ./custom_mappings_schema.json plexanisync:/plexanisync`
2. Update the script to get season 0 from plex. You just need to change a sign from > to >= in plexmodule.py. This is very easy to do with sed in the docker container: `docker exec -dit plexanisync sh -c "sed -i 's/season.seasonNumber > 0/season.seasonNumber >= 0/g' /plexanisync/plexanisync/plexmodule.py"`

Now restart the container and season 0 mappings will work. If you remove the container or the image gets updated, make sure to redo them both.

**Comment by Tahuwa** [comment]
I've been changing the files to make this work, but it would be really nice if this could be implemented by default

**Comment by Soitora** [comment]
Maybe require that specials are mapped to an AniList ID as per a single specials episode `4`, or episode range `4-6`? Instead of using the `start` feature here
